cfme/tests/services/test_service_customer_bz.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.provider import InfraProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+pytestmark = [
+    pytest.mark.meta(server_roles="+automate"),
+    pytest.mark.usefixtures('setup_provider', 'catalog_item', 'uses_infra_providers'),
+    test_requirements.service,
+    pytest.mark.long_running,
+    pytest.mark.provider([InfraProvider], selector=ONE_PER_TYPE,
+                         required_fields=[['provisioning', 'template'],
+                                          ['provisioning', 'host'],
+                                          ['provisioning', 'datastore']],
+                         scope="module"),
+]
+
+
+@pytest.mark.rhv3
+@pytest.mark.tier(2)
+def test_edit_bundle_entry_point(appliance, provider, catalog_item, request):
```

Comment:
I think there are better ways to indicate customer BZ driven tests than separate test modules.  Is there any other motivation between a separate test module instead of this test function being in cfme/tests/services/test_catalog_item.py?

Reply:
@mshriver : I created a separate module because we have started working on other service customer BZ's so I thought they can also be added here . All other service files already have lot of tests .

Reply:
I think this is ultimately going to cause confusion, and make it more difficult to locate and organize test functions. I'm willing to merge this as-is, but I think you and @niyazRedhat should really consider how these modules are organized, and what goes where. Maybe split catalog items and catalog bundles into separate modules to keep them a manageable size. Ultimately this is a pretty basic CRUD test for catalog bundle.